### PR TITLE
Add "How to bridge tokens" pages

### PR DESCRIPTION
**Pull request by TucksonDev**
This PR adds a series of How-tos focused on token bridging:

- [An overview](https://nitro-docs-git-add-how-to-bridge-tokens-offchain-labs.vercel.app/devs-how-tos/bridge-tokens/how-to-bridge-tokens-overview)
- [How to bridge tokens via Arbitrum's standard gateway](https://nitro-docs-git-add-how-to-bridge-tokens-offchain-labs.vercel.app/devs-how-tos/bridge-tokens/how-to-bridge-tokens-standard)
- [How to bridge tokens via Arbitrum's custom generic gateway](https://nitro-docs-git-add-how-to-bridge-tokens-offchain-labs.vercel.app/devs-how-tos/bridge-tokens/how-to-bridge-tokens-custom-generic)
- How to bridge tokens via a custom gateway (not done yet, waiting for confirmation)

This PR also needs this [tutorials PR](https://github.com/OffchainLabs/arbitrum-tutorials/pull/102) to be merged as it holds modifications to those scripts.

### CODE REVIEW

File: arbitrum-docs/devs-how-tos/bridge-tokens/how-to-bridge-tokens-standard.mdx [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+- uint256 `_gasPriceBid`: Gas price for the execution in L2
+- bytes `_data`: 2 pieces of data encoded:
+    - uint256 `maxSubmissionCost`: Max gas deducted from user's L2 balance to cover base submission fee
+    - bytes `extraData`: “0x”
+
+## Step 5: Wait for execution in L2
+
+After calling the deposit method (or the outboundTransferCustomRefund, if you’re going the manual way), you’ll have to wait a bit until the message is executed in L2. We will verify the status of the underlying retryable ticket created to bridge the tokens. Check this page, to know more about [L1-to-L2 messages, also known as retryables](/arbos/l1-to-l2-messaging.mdx).
+
+You can programmatically wait for the execution of the transaction in L2 using Arbitrum’s SDK. You should first wait for the execution of the submission transaction (the one sent to the router contract) and then the execution of the L2 transaction.
```
Comment (mahsamoosavi) [inline arbitrum-docs/devs-how-tos/bridge-tokens/how-to-bridge-tokens-standard.mdx:185]:
```suggestion
You can programmatically wait for the execution of the transaction on L2 using Arbitrum’s SDK. You should first wait for the execution of the submission transaction (the one sent to the router contract) and then the execution of the L2 transaction.
```

File: arbitrum-docs/devs-how-tos/bridge-tokens/how-to-bridge-tokens-standard.mdx [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+l2Result.complete
+  ? console.log(
+      `L2 message successful: status: ${L1ToL2MessageStatus[l2Result.status]}`
+    )
+  : console.log(
+      `L2 message failed: status ${L1ToL2MessageStatus[l2Result.status]}`
+    )
+```
+
+If you’re going the manual way, you can verify if the message has been executed in L2 through the [Retryables Dashboard](https://retryable-dashboard.arbitrum.io/). Just paste the hash of transaction submitted to the router contract and the tool will tell you whether it’s been redeemed or not.
```
Comment (mahsamoosavi) [inline arbitrum-docs/devs-how-tos/bridge-tokens/how-to-bridge-tokens-standard.mdx:206]:
```suggestion
If you’re going the manual way, you can verify if the message has been executed on L2 through the [Retryables Dashboard](https://retryable-dashboard.arbitrum.io/). Just paste the hash of transaction submitted to the router contract and the tool will tell you whether it’s been redeemed or not.
```

File: arbitrum-docs/devs-how-tos/bridge-tokens/how-to-bridge-tokens-standard.mdx [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+
+## Step 6: Check the new token contract created in L2
```
Comment (mahsamoosavi) [inline arbitrum-docs/devs-how-tos/bridge-tokens/how-to-bridge-tokens-standard.mdx:208]:
```suggestion
## Step 6: Check the new token contract created on L2
```

File: arbitrum-docs/devs-how-tos/bridge-tokens/how-to-bridge-tokens-standard.mdx [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+
+Finally, let’s find the token contract that has been created in L2.
```
Comment (mahsamoosavi) [inline arbitrum-docs/devs-how-tos/bridge-tokens/how-to-bridge-tokens-standard.mdx:210]:
```suggestion
Finally, let’s find the token contract that has been created on L2. 
```

File: arbitrum-docs/devs-how-tos/bridge-tokens/how-to-bridge-tokens-standard.mdx [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+    )
+```
+
+If you’re going the manual way, you can verify if the message has been executed in L2 through the [Retryables Dashboard](https://retryable-dashboard.arbitrum.io/). Just paste the hash of transaction submitted to the router contract and the tool will tell you whether it’s been redeemed or not.
+
+## Step 6: Check the new token contract created in L2
+
+Finally, let’s find the token contract that has been created in L2. 
+
+Using Arbitrum’s SDK, you can call method [getL2ERC20Address](/sdk/classes/assetBridger_erc20Bridger.Erc20Bridger#getl2erc20address) of the  [ERC20Bridger](/sdk/classes/assetBridger_erc20Bridger.Erc20Bridger) class, which will return the address of the token contract in L2 which corresponds to the L1 token contract send as parameter.
```
Comment (mahsamoosavi) [inline arbitrum-docs/devs-how-tos/bridge-tokens/how-to-bridge-tokens-standard.mdx:212]:
```suggestion
Using Arbitrum’s SDK, you can call method [getL2ERC20Address](/sdk/classes/assetBridger_erc20Bridger.Erc20Bridger#getl2erc20address) of the [ERC20Bridger](/sdk/classes/assetBridger_erc20Bridger.Erc20Bridger) class, which will return the address of the token contract in L2 that corresponds to the L1 token contract sent as parameter.
```

**Review by mahsamoosavi — CHANGES_REQUESTED** [review]
A few changes and then it looks good to me 💙

File: arbitrum-docs/devs-how-tos/bridge-tokens/how-to-bridge-tokens-custom-generic.mdx [hunk 6]
```diff
@@ -0,0 +1,384 @@
...
+description: Learn how to use the custom generic gateway to bridge tokens programmatically
+reader_audience: developers who want to build on Ethereum/Arbitrum and bridge tokens between layers
+content_type: how-to
+---
+
+import PublicPreviewBannerPartial from '../../partials/_public-preview-banner-partial.md'; 
+
+<PublicPreviewBannerPartial />
+
+In this how-to you’ll learn how to bridge your own token between Ethereum (Layer 1 or L1) and Arbitrum (Layer 2 or L2), using [Arbitrum’s custom generic gateway](/asset-bridging.mdx#the-arbitrum-generic-custom-gateway). For alternative ways of bridging tokens, don’t forget to check out this [overview for token bridging](/devs-how-tos/bridge-tokens/how-to-bridge-tokens-overview.mdx).
```
[the remark this answers is not in the document]
Comment (TucksonDev) [reply arbitrum-docs/devs-how-tos/bridge-tokens/how-to-bridge-tokens-custom-generic.mdx:13]: In this part I'm referring to "token" as their contract. Shouldn't that be in singular form?

File: arbitrum-docs/devs-how-tos/bridge-tokens/how-to-bridge-tokens-custom-generic.mdx [hunk 7]
```diff
@@ -0,0 +1,384 @@
...
+
+## Step 3: Register the custom token to the custom generic gateway
+
+Once both our contracts are deployed in their respective chains, it’s time to register the token in the custom generic gateway.
+
+As mentioned before, this action needs to be done by the L1 token, and we’ve implemented the function `registerTokenOnL2` to do it. So now we only need to call that function.
+
+When using this function two actions will be performed:
+
+1. Call function `registerTokenToL2` of `L1CustomGateway`. This will change the `l1ToL2Token` internal mapping it holds and will send a retryable ticket to the counterpart `L2CustomGateway` contract in L2, to also set its mapping to the new values.
```
[the remark this answers is not in the document]
Comment (mahsamoosavi) [reply arbitrum-docs/devs-how-tos/bridge-tokens/how-to-bridge-tokens-custom-generic.mdx:305]: yeah not sure either, I just suggested this as I found you don't use dots in similar situations here, doesn't really matter as long as we are consistent

**Review by mahsamoosavi — APPROVED** [review]

**Pull request merged**